root-packages/libgpiod/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git
+TERMUX_PKG_DESCRIPTION="C library and tools for interacting with the GPIO character device. Requires kernel v4.6 or higher"
```

Comment:
According to https://cateee.net/lkddb/web-lkddb/GPIO_CDEV.html, the CONFIG_GPIO_CDEV is available from Linux v5+

Reply:
I am using 4.19.x kernel which does not have CONFIG_GPIO_CDEV enabled but `/dev/gpiochip*` are present.

Reply:
Ah, right, my description is bad (and should be updated). The package needs [this commit](https://github.com/torvalds/linux/commit/ff2b1359229927563addbf2f5ad480660c350903) which landed is 4.6. After [this commit](https://github.com/torvalds/linux/commit/d143493c01b7fc3a9b9369a795e20329561222c5) which landed in 5.10 it was split out into a separate config option GPIO_CDEV.